README.md

```diff
@@ -1,4 +1,4 @@
-# Getting Started with Create React App
+# Getting Started with Create React
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
```

Comment:
- [ ] Kindly use the Microverse professional [README Template](https://github.com/microverseinc/readme-template). Currently, your README is the default README generated with the create-react-app command but you're to use the Microverse README template as required in the project requirements.